app/views/devise/registrations/edit.html.erb

```diff
@@ -1,43 +1,32 @@
-<h2>Edit <%= resource_name.to_s.humanize %></h2>
-
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
-  <%= devise_error_messages! %>
-
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-  </div>
-
-  <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
-    <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
+<div class = "profile_show_and_edit">
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+    <%= devise_error_messages! %>
+
+    <div class="tabbed">
+      <div class="label">user:    </div>
+      <%= @user.name %>
+    </div>
+    <div class="tabbed">
+      <div class="label">created:    </div>
+      <%= @user.created_at %>
+    </div>
+    <div class="tabbed">
+      <div class="label">karma:</div>
+      <div>TBD</div>
+    </div>
+
+    <div class="field tabbed">
+      <div class="label">about:</div>
+      <%= f.text_area :about, :rows => 4 %>
```

Comment:
since the new Hash syntax is used elsewhere, I'd suggest updating this to read `rows: 4`